File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.java
class Solution {
    public int[] solution(int[] numbers, String direction) {
        int[] answer = new int[numbers.length];
        if(direction.equals("right")){
            for(int i=0;i<numbers.length;i++){
                answer[(i!=numbers.length-1)?(i+1):0]=numbers[i];
            }
        }
        else if(direction.equals("left")){
            for(int i=0;i<numbers.length;i++){
                answer[(i!=0)?(i-1):numbers.length-1]=numbers[i];
            }
        }
        return answer;
}
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    answer = [0]*len(numbers)
    if direction=="right":
        for i in range(0,len(numbers)):
            answer[i]=numbers[i-1]
    elif direction=="left":
        for i in range(0,len(numbers)-1):
            answer[i]=numbers[i+1]
        answer[len(numbers)-1]=numbers[0]
    return answer